src/SearchWrapper.cpp

```diff
@@ -197,9 +197,102 @@ parseQuery(Rcpp::List & query)
 		}
 	    }
 	}
-
+      
+#else
+      
+      //check for valueRangeProcessors
+      if(query.containsElementNamed("VRP")){ 
+        Rcpp::List processors=query["VRP"];
+        
+        for( Rcpp::List::iterator it = processors.begin(); it != processors.end(); ++it ) {
+          Rcpp::List list=it.operator*();
+          string type=Rcpp::as<std::string>(list["type"]);
+          
+          if(!type.compare("proc.num")){ 
+            int aVal=list["value"];
+            if(list.containsElementNamed("check.str")){
+              string aStr=Rcpp::as<std::string>(list["check.str"]);
+              
+              bool aPrefix=true;
+              
+              Rcpp::StringVector flagVec;
+              if (list.containsElementNamed("flags")) {
+                flagVec = list["flags"];
+                for (Rcpp::StringVector::iterator itr = flagVec.begin(); itr != flagVec.end(); ++itr) {
+                  std::string aflag = Rcpp::as<std::string>(*itr);
+                  if ((!aflag.compare("RP_SUFFIX")) || (!aflag.compare("RP_REPEATED")))
+                        aPrefix=false;
+                }
+              }
+              
+              Xapian::NumberValueRangeProcessor *num_proc=new Xapian::NumberValueRangeProcessor(aVal,aStr,aPrefix);
+              queryparser.add_valuerangeprocessor(num_proc);
+            }
+            else{
+              Xapian::NumberValueRangeProcessor *num_proc=new Xapian::NumberValueRangeProcessor(aVal);
+              queryparser.add_valuerangeprocessor(num_proc);
+            }
+          }
+          if(!type.compare("proc.str")){
+            int aVal=list["value"];
+            if(list.containsElementNamed("check.str")){
+              string aStr=Rcpp::as<std::string>(list["check.str"]);
+              bool aPrefix=true;
+              
+              Rcpp::StringVector flagVec;
+              if (list.containsElementNamed("flags")) {
+                flagVec = list["flags"];
+                for (Rcpp::StringVector::iterator itr = flagVec.begin(); itr != flagVec.end(); ++itr) {
+                  std::string aflag = Rcpp::as<std::string>(*itr);
+                  if ((!aflag.compare("RP_SUFFIX")) || (!aflag.compare("RP_REPEATED")))
```

Comment:
If I pass a totally different flag string, I get no error and no prefix support. If I pass no flags at all, I get prefix support. I think this is the wrong way round: set `aPrefix` to `false`, then if you find `RP_SUFFIX` in the flags, set it to `true`.

I think you should ignore `RP_REPEATED` with 1.2, or that should be an error. Xapian 1.2 doesn't have similar functionality — and if you want it to work to with 1.4 you need to pass both: `RP_SUFFIX | RP_REPEATED`, ie `c('RP_SUFFIX', 'RP_REPEATED')`.